temporian/core/event_set_ops.py

```diff
@@ -1925,6 +1925,52 @@ def select(
 
         return select(self, feature_names=feature_names)
 
+    def select_index_values(
+        self: EventSetOrNode,
+        keys: Optional[Union[IndexKey, List[IndexKey]]] = None,
```

Comment:
Do we want to support passing indexes that only contain one item without a tuple?
e.g. select_index_values(["A", "B"])

Reply:
changed `IndexKey = Union[Tuple[IndexValue, ...], IndexValue]` to support receiving single value